cmake/std/trilinosprhelpers/setenvironment/SetEnvironment.py

```diff
@@ -403,15 +414,11 @@ def _load_configuration_r(self, config_section, actions=None, processed_secs=Non
         if processed_secs is None:
             processed_secs = {}
 
-        # Exit if we've already processed this section (recursion breaker)
-        if config_section in processed_secs.keys():
-            print("WARNING: Cyclic dependencies encountered when loading {} ... {}".format(self.profile, config_section))
-            return actions
-
+        print("Section: {}".format(config_section))
         processed_secs[config_section] = True
 
         for op2,value in sec.items():
-            #print("> k=`{}` v=`{}`".format(k,v))
+            #print(": op2=`{}` value=`{}`".format(op2,value))
```

Comment:
Do you want to remove this commented out line?

Reply:
no. It's useful for debugging stuff... easier to just uncomment when I'm tracking down bugs and want to trace what its' doing.